infra/aws/terraform/prow-build-cluster/terraform.canary.tfvars

```diff
@@ -35,10 +35,14 @@ cluster_autoscaler_version = "v1.25.0"
 # Ubuntu EKS optimized AMI: https://cloud-images.ubuntu.com/aws-eks/
 node_ami            = "ami-07e8e7dddc8b3bad9"
 node_instance_types = ["r5d.xlarge"]
+
+node_ami_green            = "ami-07e8e7dddc8b3bad9"
+node_instance_types_green = ["r5d.xlarge"]
+
 node_volume_size    = 100
 
 node_min_size                   = 1
-node_max_size                   = 3
+node_max_size                   = 4
```

Comment:
That was me temporary fiddling with instances while testing AlertManager, we can revert this to 3.